src/helpers/responseHandler.js

```diff
@@ -11,8 +11,11 @@
   * @param {Object} additionalFields
   * @returns {Object} null
   */
-export const respondWithSuccess = (res, statusCode = 200, message, additionalFields = {}) => res
-  .status(statusCode).send({ success: true, message, payload: { ...additionalFields } });
+export const respondWithSuccess = (res, statusCode = 200, message, additionalFields = {}) => {
+  additionalFields = Array.isArray(additionalFields) ? [ ...additionalFields ] : { ...additionalFields}
+  res.status(statusCode).send({ success: true, message, payload: additionalFields });
+} 
```

Comment:
Please, can you use the previous code and try to work around it. Thanks.

Reply:
@joaquinto It's fine. this is account for when the payload is an array.